src/api/changelogInfo.js

```diff
@@ -26,3 +27,20 @@ export function saveChangelogInfo(
     jsYaml.safeDump(newChangelogInfo)
   );
 }
+
+function ensureInitializedProject(config) {
+  const hasEntries = globSync(joinPath(config.path, '**/*')).length > 0;
+
+
+  if (existsSync(getInfoFilePath(config))) {
+    return;
+  }
+
+  saveChangelogInfo(config, {
+    version: hasEntries ? -1 : CURRENT_VERSION
```

Comment:
Hmm, I'd vote to document these cases with comments.
I think without the PR description (just from the code) it's not super clear WHEN one of these two cases would happen. I.e.:
- `info.yml` missing BUT entries present  => Must be a project on an old strangelog version without info.yml => needs to migrate from CURRENT_VERSION
- `info.yml` missing and NO entries present => Create `info.yml`, since it must be a new project.

Hmm, thinking about it, why is the `info.yml` not created as part of a migration, but here instead?